### PR TITLE
Add cluster sync to cli 

### LINKED ISSUE #815: SPK introspection tool should visualize cluster and ring information
**As a:** user of spk

**I want:** to visualize the two dimensional setup of my clusters and rings

**So that:** I can understand what versions of services are running on a particular ring in a particular cluster/environment.

**Describe the solution you'd like:**
The following pieces are a nice to have for the dashboard:
- Show some sort of indication that there are multiple cluster regions watching the same manifest repository
- Indicate the region names based on their flux sync tags, such as `flux-usa`, `flux-emea` and display the deployment running on these clusters on the dashboard, perhaps by showing multiple cluster sync indicators in a column (need some work with the designer)
- Brainstorm and come up with some user experiences that may be a good solution for fitting this within the current dashboard, or possibly consider adding a separate view to visualize this information.

An example of a service matrix formed by clusters and rings:

| Region/Ring     | (... custom rings) | Dev | QA | Prod |
| ---      | ---       | ---      | ---       | --- |
| USA | 0.1         | 0.2         | 0.1         | 0.1         | 
| Europe     | 0.1        | 0.2        | 0.2         | 0.1         |
| Africa     | 0.1        | 0.2        | 0.1         | 0.1         |

Related work: 
- [x] [Introspection tool should read multiple flux sync tags prefixed by `flux`](https://github.com/microsoft/bedrock/issues/814)
- [x] [Add git-sync-tag variable to terraform template for flux](https://github.com/microsoft/bedrock/issues/813)
- [x] [Introspection tool should display multiple cluster sync statuses in a separate column](https://github.com/microsoft/bedrock/issues/840)
- [ ] `spk deployment` commands should have support for cluster sync statuses #844

**Acceptance Criteria:**
A solution for cluster information along with rings

**Describe alternatives you've considered:**

**Additional context:**

**Does this require updates to documentation?:** Yes

### LINKED ISSUE #844: [spk] Introspection CLI should have support for cluster sync statuses
**As a:** user

**I want:** to query via spk cli the cluster sync status of my deployment

**So that:** I can verify that my service is running on the most recent manifest commit

**Describe the solution you'd like:**
Right now, there's support for cluster sync on the dashboard but not in the CLI. 
- [x] Add support for cluster sync feature by supporting it in the CLI just like it's supported in the dashboard.

**Acceptance Criteria:**
User can get cluster sync information without going to dashboard

**Describe alternatives you've considered:**
User would have to launch the dashboard to get this information.

**Additional context:**

**Does this require updates to documentation?:**

### PR DESCRIPTION
**Pull request by samiyaakhtar**
Closes https://github.com/microsoft/bedrock/issues/844 
Closes https://github.com/microsoft/bedrock/issues/815

Screen Shot 2020-02-12 at 12 55 10 PM

### CODE REVIEW

**Comment by andrebriggs** [comment]
@samiyaakhtar can you add a screenshot of the shell experience with cluster sync in the output now?

**Comment by samiyaakhtar** [comment]
@andrebriggs added a [screenshot](https://user-images.githubusercontent.com/1795990/74376409-eaec2880-4d96-11ea-874f-5a44ddadf5bd.png)

File: src/commands/deployment/get.ts [hunk 1]
```diff
@@ -142,43 +146,46 @@ export const processOutputFormat = (outputFormat: string): OUTPUT_FORMAT => {
...
- * @param imageTag docker image tag name
- * @param p1Id identifier of the first build pipeline (src to ACR)
- * @param commitId commit Id into the source repo
- * @param service name of the service that was modified
- * @param deploymentId unique identifier for the deployment
+ * @param initObj captures keys and objects during the initialization process
+ * @param values validated command line values
  */
-export const getDeployments = (
+export const getDeployments = async (
```
Comment (dennisseah) [inline src/commands/deployment/get.ts]: `async` is not needed because this function is already returning a Promise

**Review by dennisseah — APPROVED** [review]

File: src/commands/deployment/get.ts (hunk 1 above)
Comment (samiyaakhtar) [reply src/commands/deployment/get.ts]: removed

**Review by andrebriggs — APPROVED** [review]

**Pull request merged**